### ISSUE TITLE
Блокиране на телевизия Алфа

### ISSUE THREAD

**Issue by kunev** [openmediabg/dancewithme#25]
Предвид липсата на докумкентално доказуема икономическа връзка между телевизия Алфа и г-н Пеевски, редно ли е сайтът им(атака дот тиви) да бъде включен в списъка с блокирани домейни?

**Comment by ignisf** [comment]
Не знам.. Обективно ли отразяват събитията?

**Comment by mitio** [comment]
Евгени, `ataka.tv` не е в списъка с блокирани домейни?

**Comment by kunev** [comment]
Да, не е в списъка. Сега се замислям, че може би не съм си формулирал питането правилно. _Предлагам_ да бъде добавена, но имаме ли доказуеми връзки с &lt;insert-random-geometrical-figure-here&gt; на Пеевски?

Иначе липсата на обективност и журналиситческа етика от каквото и да било естество се вижда без много усилия.
